applications/incompressible_navier_stokes/navier_stokes_manufactured/application.h

```diff
@@ -0,0 +1,798 @@
+
+    // viscous step
+    this->param.solver_viscous         = SolverViscous::CG;
+    this->param.solver_data_viscous    = SolverData(1000, 1.e-12, 1.e-8);
+    this->param.preconditioner_viscous = PreconditionerViscous::InverseMassMatrix; // Multigrid;
+
+
+    // PRESSURE-CORRECTION SCHEME
+
+    // formulation
+    this->param.order_pressure_extrapolation = this->param.order_time_integrator - 1;
+    this->param.rotational_formulation       = true;
+
+    // momentum step
+
+    // Newton solver
+    this->param.newton_solver_data_momentum = Newton::SolverData(100, 1.e-14, 1.e-6);
+
+    // linear solver
+    this->param.solver_momentum                = SolverMomentum::GMRES;
+    this->param.solver_data_momentum           = SolverData(1e4, 1.e-12, 1.e-8, 100);
+    this->param.preconditioner_momentum        = MomentumPreconditioner::InverseMassMatrix;
+    this->param.update_preconditioner_momentum = true;
+
+
+    // COUPLED NAVIER-STOKES SOLVER
+
+    // nonlinear solver (Newton solver)
+    this->param.newton_solver_data_coupled = Newton::SolverData(100, 1.e-10, 1.e-6);
+
+    // linear solver
+    this->param.solver_coupled      = SolverCoupled::FGMRES;
+    this->param.solver_data_coupled = SolverData(1e4, 1.e-12, 1.e-8, 100);
+
+    // preconditioning linear solver
+    this->param.preconditioner_coupled        = PreconditionerCoupled::BlockTriangular;
+    this->param.update_preconditioner_coupled = true;
+
+    // preconditioner velocity/momentum block
+    this->param.preconditioner_velocity_block = MomentumPreconditioner::Multigrid;
+
+    if(this->param.treatment_of_convective_term == TreatmentOfConvectiveTerm::Implicit &&
+       include_convective_term == true)
+      this->param.multigrid_operator_type_velocity_block =
+        MultigridOperatorType::ReactionConvectionDiffusion;
+    else
+      this->param.multigrid_operator_type_velocity_block = MultigridOperatorType::ReactionDiffusion;
+
+    this->param.multigrid_data_velocity_block.smoother_data.smoother =
+      MultigridSmoother::Chebyshev; // Jacobi; //Chebyshev; //GMRES;
+    this->param.multigrid_data_velocity_block.smoother_data.preconditioner =
+      PreconditionerSmoother::PointJacobi; ////BlockJacobi;
+    this->param.multigrid_data_velocity_block.smoother_data.iterations        = 5;
+    this->param.multigrid_data_velocity_block.smoother_data.relaxation_factor = 0.7;
+    this->param.multigrid_data_velocity_block.coarse_problem.solver =
+      MultigridCoarseGridSolver::GMRES;
+
+    // preconditioner Schur-complement block
+    if(this->param.treatment_of_convective_term == TreatmentOfConvectiveTerm::Explicit)
+      this->param.preconditioner_pressure_block = SchurComplementPreconditioner::CahouetChabard;
+    else
+      this->param.preconditioner_pressure_block =
+        SchurComplementPreconditioner::PressureConvectionDiffusion;
+  }
+
+  void
+  create_grid() final
+  {
+    dealii::GridGenerator::hyper_cube(*this->grid->triangulation,
```

Comment:
Please add a comment that this does not follow the (new) way of creating grids for global-coarsening multigrid with fully-distributed tria.